marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/load/loadPageValidations.cy.tsx

```diff
@@ -0,0 +1,60 @@
+import {toolbar} from "../../../support/components/common";
+import LoginPage from "../../../support/pages/login";
+import loadPage from "../../../support/pages/load";
+import "cypress-wait-until";
+
+describe("Custom Ingestion", () => {
```

Comment:
Instead of custom ingestion we can change it to "Editing a Load step and no other step is affected"
Or describe be "Editing a load step" and it be "Verify other load steps do not change". Or something along those lines.